en/developer_guide/multiple_device.rst

```diff
@@ -0,0 +1,89 @@
+Multiple device
+===============
+
+Prerequisite
+------------
+
+To enable the multi-device rendering, we assume that the server in front of Php (Varnish, Apache2, Nginx, ...)
+ implements the Wurfl librairy and is able to add a request header with the device type.
+
+The device name will be used in the configuration and in the template name following the rule :
+
+* main template : template.html.twig
+* phone template : template.phone.html.twig
+
+Configuration
+-------------
+
+To activate the adaptative response, you have to define all the devices name in the configuration :
+
+.. code-block:: yaml
+
+    #app\config\config.yml
+    open_orchestra_front:
+        devices:
+            web: ~
+            tablet:
+                parent: web
+            phone:
+                parent: web
+            android:
+                parent: phone
```

Comment:
Are these devices names hard coded ? If so, maybe make it clear before. Can I add my own device name ? How ?